src/jit/bitsetasuint64inclass.h

```diff
@@ -180,11 +180,10 @@ class BitSetUint64
 
     inline void ClearD(Env env)
     {
-        // Recall that ClearD does *not* require "*this" to be of the current epoch.
+        // Recall that ClearD requires "*this" to be of the current epoch.
         Uint64BitSetOps::ClearD(env, m_bits);
```

Comment:
Given that this is changing the requirements on ClearD, have you examined all the callsites to ensure that they are consistent with this? I would suggest that this undergo additional testing (e.g. desktop SPMI) to ensure that we don't encounter this assert.